helm/rbac-operator/templates/configmap.yaml

```diff
@@ -22,7 +22,6 @@ data:
           caFile: ''
           crtFile: ''
           keyFile: ''
-      namespaceAuth:
-        tenantAdminTargetGroup: {{ .Values.Installation.V1.Kubernetes.Auth.TenantAdminTargetGroup }}
-        viewAllTargetGroup: {{ .Values.Installation.V1.Kubernetes.Auth.ViewAllTargetGroup }}
+      writeAllCustomerGroup: {{ .Values.oidc.customer.admin_group }}
+      writeAllGiantswarmGroup: {{ .Values.oidc.giantswarm.admin_group}}
```

Comment:
Why do we call it here `wrieAll` and `admin` in values?

How did it work before with this flag https://github.com/giantswarm/rbac-operator/blob/0585c73553a6ef9cfb680f8464d24b1441259ad5/flag/service/service.go#L11 ?

Reply:
It is just naming. it has nothing to do with actual logic. But renamed for sanity